stdlib/public/Compatibility50/Overrides.h

```diff
@@ -0,0 +1,30 @@
+//===--- Overrides.cpp - Compat overrides for Swift 5.0 runtime ----s------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2017 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+//
+//  This file provides compatibility override hooks for Swift 5.0 runtimes.
+//
+//===----------------------------------------------------------------------===//
+
+#include "swift/Basic/LLVM.h"
+#include "swift/Runtime/Metadata.h"
+
+namespace swift {
+
+using ConformsToProtocol_t =
+  const WitnessTable *(const Metadata *, const ProtocolDescriptor *);
+
+const WitnessTable *
+swiftoverride_conformsToProtocol(const Metadata * const type,
```

Comment:
You probably shouldn't give this an exported symbol at all.

Reply:
It should be hidden visibility by default. We build with `-fvisibility=hidden`.

Reply:
Sure, but since it's a static library…it can be even better. Even if you're supposed to only get one copy of it.

Reply:
I suppose I could lump the whole thing into one .o file, but that wouldn’t scale very well if we add more back deployment stuff. What did you have in mind?

Reply:
Hm. I had been thinking that you'd have a different override section for each entry point, and then none of them need to be exposed, but maybe that wouldn't work, or maybe it'd be a clunky solution anyway. I guess it's fine this way.

Reply:
It would be a good idea to more strongly namespace these as `swift50override_`, in case a future 5.1 compatibility library wants to provide a different hook for the same entry point.